lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+  attr_accessor :cells
+
+  def initialize()
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    if @cells[input.to_i] == "O" || @cells[input.to_i] == " " || @cells[input.to_i] == "X"
+     @cells[input.to_i-1]
+   end
+  end
+
+  def full?
+    @cells.any? {|space| space == " "} ? false : true
+  end
+
+  def taken?(input)
+    if @cells[input.to_i - 1] == "X" || @cells[input.to_i - 1] == "O"
```

Comment:
we could remove this if statement and it would implicitly return true or false already.